reporter/src/main/kotlin/reporters/ExcelReporter.kt

```diff
@@ -92,6 +97,11 @@ class ExcelReporter : Reporter {
         outputDir: File,
         options: Map<String, String>
     ): List<File> {
+        val extraColumns = options[OPTION_EXTRA_COLUMNS].orEmpty()
+            .split(",")
+            .map { it.trim() }
```

Comment:
Is this really required? The options (usually) come from command line options, and if these had spaces, they would be different options.

Reply:
I wasn't sure if there are any other whitespace characters that could somehow be passed here, so I thought better make sure there are none left.